src/assets/i18n/en.json

```diff
@@ -2,6 +2,11 @@
     "APP": {
         "BROWSER_NOT_SUPPORTED": "Your browser is not supported, please download chrome 57 or higher"
     },
+    "TOP_BAR": {
+        "CREATE_VIDEO": "Create a video",
```

Comment:
In my opinion, "Upload a video" is better description then "Create a video" 
Because you dont really create the video, you upload it.